src/Pages/Wallets.razor

```diff
@@ -450,7 +450,7 @@
                     <Column ColumnSize="ColumnSize.Is6">
                         <Field>
                             <FieldBody>
-                                <Check TValue="bool" @bind-Checked="@_transferAllFunds">Transfer all funds</Check>
+                                <Check TValue="bool" CheckedChanged="OnTransferAllFundsCheckedChanged" Checked="@_transferAllFunds">Transfer all funds</Check>
```

Comment:
With this function added `Checked="@_transferAllFunds"` doesn't do anything

Reply:
what do you mean? I'm using the manual event binding as in the docs https://blazorise.com/docs/components/check

Reply:
Yes, its just a small thing, that the assignation is being duplicated:
```
private void OnTransferAllFundsCheckedChanged(bool value)
    {
        _transferAllFunds = value;
        _amountToTransfer = _sourceBalance;
    }
```